src/test/java/test/SynchronyTest.java

```diff
@@ -39,6 +40,19 @@ void synchrony_enable(Product product) throws Exception {
                 .hasTextNotContaining("synchrony.btf.disabled");
     }
 
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void synchrony_disabled(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "synchrony.enabled", "false"
+        ));
+
+        final var sysProps = resources.get(Kind.ConfigMap, product.getHelmReleaseName() + "-jvm-config")
+                .getNode("data", "additional_jvm_args");
+
+        assertThat(sysProps).hasTextContaining("synchrony.btf.disabled");
```

Comment:
Because Synchrony is now disabled by default for the comparison test, I've added this test to verify that the property is added if we disable synchrony.